### ISSUE TITLE
Document best practices for page titles

### ISSUE THREAD

**Issue by blackbaud-johnly** [blackbaud/skyux2-docs#280]
Based on @Blackbaud-MattGregg's docs on navigation guidelines, create new topic in the Design section about guidance for page titles. Here's the notes form Matt's docs:

General pattern for <title></title>element:[Page title] –Product titleE.g., [Page title] Guidelines

- Keep it short

- Use a concise description of the contents or purpose of the page which can stand on its own and make sense out of context

- Put most meaningful, differentiating terms at the front so they can be scanned in tab labels, window lists, bookmarksEach page title must be unique, do not repeat page titles across pages

- Do not include product namesor “Blackbaud”in the [page title]. These will be added to the <title> element by SKY UX depending on the context.

**Comment by Blackbaud-MattGregg** [comment]
@blackbaud-johnly Just an FYI this content is currently on the page design page in the design guidelines https://developer.blackbaud.com/skyux/design/guidelines/page-layouts , but I'd expect this to go on the new Editorial/Content guidelines that Paula is working on when we get to launching new page

**Comment by blackbaud-johnly** [comment]
Closing as this is already documented in the page design guidelines. We can circle back to including it in the content guidelines when we add those.
